pkg/controller/pxc/full_crash_recovery.go

```diff
@@ -0,0 +1,152 @@
+package pxc
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	v1 "github.com/percona/percona-xtradb-cluster-operator/pkg/apis/pxc/v1"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	ErrNotAllPXCPodsRunning = errors.New("not all pxc pods are running")
+	logLinesRequired        = int64(1)
```

Comment:
We shouldn't rely only on the last line, should we? I don't know if it's suitable to grind thru all log lines (it might be huge I suppose), but I suggest checking at least last 5-10 lines or so (you know better what could and couldn't be in a log)

Reply:
d89611f